From 245ef6cb85361a914dc4d303023eeea6b4a72833 Mon Sep 17 00:00:00 2001
From: Oliver Talevski <oliver.talevski@esss.se>
Date: Thu, 22 Nov 2018 08:03:55 +0100
Subject: [PATCH] Set CONFIG_FSL_IFC default y

	Otherwise it is disabled is ifc nand is disabled but we also
	have the PON fpga on the same bus.
---
 drivers/memory/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/memory/Kconfig b/drivers/memory/Kconfig
index 820f559..70c6d6d 100644
--- a/drivers/memory/Kconfig
+++ b/drivers/memory/Kconfig
@@ -115,6 +115,7 @@ config FSL_CORENET_CF
 
 config FSL_IFC
 	bool
+	default y
 	depends on FSL_SOC || ARCH_LAYERSCAPE || SOC_LS1021A
 
 config JZ4780_NEMC
-- 
2.7.4

